test/purging.sh.in

```diff
@@ -134,24 +134,45 @@ echo " == no matter when they were deleted"
 $RMW_TEST_CMD_STRING --verbose $HOME/somefiles
 test_result $?
 
-# echo " == Should not work if '-f' isn't used"
-# output=`echo "y" | $RMW_ALT_TEST_CMD_STRING --purge -e`
-# test "$output" = "purge has been skipped: use -f or --force"
-# test_result $?
-# 
+echo " == Should not work if '-f' isn't used"
+output=`echo "y" | $RMW_ALT_TEST_CMD_STRING --purge -e`
+test "$output" = "purge has been skipped: use -f or --force"
+test_result $?
+
 # echo " == Should not work if '--purge' isn't used"
-# output=`echo "y" | $RMW_TEST_CMD_STRING -ffe`
+# output=`echo "y" | $RMW_ALT_TEST_CMD_STRING -ffe`
 # test "$output" = "'-f/--force' does nothing without '-g/--purge'"
 # test_result $?
-#
-echo " == empty works with force empty (-ffe)"
+
+echo " == Should not work 'Y' or 'y' is not supplied."
+output=$(echo `echo "yfw" | $RMW_TEST_CMD_STRING --purge -e`)
+test "$output" = "All the files in trash will be unrecoverable after empty. Continue? (y/n): Action cancelled."
```

Comment:
I think it's a good idea that you're testing for these extra strings like "yfw", etc. But I think we'll be better off avoiding testing output messages when we can. If we ever change the code to emit different output messages, these scripts would have to be updated.. but another reason is it's just a little more robust to check the output of ls like this:

```
echo " == (files should still be present)"
test -z $(ls -A $HOME/.trash.rmw/files)
test_result_want_fail $?
test -z $(ls -A $HOME/.trash.rmw/info)
test_result_want_fail $?
```

And as I mentioned in another comment, we can create a separate test for the user_verify function, and maybe that could just be done with a .c file (in a separate patch)

Reply:
I've added your suggested changes.